core/src/main/java/com/yahoo/imapnio/async/data/ExtensionMailboxInfo.java

```diff
@@ -15,9 +15,18 @@
     /** Literal for MAILBOXID. */
     private static final String MAILBOX_ID = "MAILBOXID";
 
+    /** Literal for CLOSED. */
```

Comment:
Missing NOMODSEQ?

Reply:
I don't need that now. if needed we can add it.